test/test-creation.js

```diff
@@ -1386,6 +1498,8 @@ describe('JHipster client-2 generator', function () {
 
         it('creates expected files for default configuration for client-2 generator', function () {
             assert.noFile(expectedFiles.server);
+            assert.noFile(expectedFiles.serverNG1);
+            assert.noFile(expectedFiles.serverNG2);
             assert.noFile(expectedFiles.maven);
             assert.file(expectedFiles.i18nJson);
             assert.file(getFilesForOptions(require('../generators/client-2/files').files, {
```

Comment:
@agoncal we dont need to maintain a list of files for ng2 and ng1. for ng2 im building the list on the fly here and soon ill do the same for ng1 as well so for now you dont need to change anything on this test files. it should work as it is